spec/integration/integration_spec_helper.rb

```diff
@@ -64,6 +65,57 @@ def normalize_inspect_output(output)
     gsub!(/(\r\033\[K.*?\r\033\[K).*\r\033\[K(.*)/, "\\1\\2") # strip all progress messages but two
 end
 
+def machinery_host(description)
+  description.sub!(/.*@/, "")
+end
+
+def boxes
+  {
+    "openSUSE_13_2" => "opensuse132",
+    "openSUSE_13_1" => "opensuse131",
+    "sles12" => "sles12",
+    "sles11" => "sles11sp3",
+    "rhel5" => "rhel5",
+    "rhel6" => "rhel6"
+  }
+end
+
+def test_group_hierarchy
+  {
+    "full" => %w(full_test minimal_test acceptance_test),
+    "minimal" => %w(minimal_test acceptance_test),
+    "acceptance" => %w(acceptance_test)
+  }
+end
+
+def matrix_path
+  File.join(Machinery::ROOT, "spec", "definitions", "support", "integration_tests.yml")
+end
+
+def include_examples_for_platform(currently_running_on)
```

Comment:
Assignment Branch Condition size for include_examples_for_platform is too high. [30.5/15]